posthog/tasks/scheduled.py

```diff
@@ -78,10 +76,12 @@ def add_periodic_task_with_expiry(
     name: str | None = None,
 ) -> None:
     """
-    If the workers get delayed in processing tasks, then tasks that fire every X seconds get queued multiple times
-    And so, are processed multiple times. But they often only need to be processed once.
-    This schedules them with an expiry so that they aren't processed multiple times.
-    The expiry is larger than the schedule so that if the worker is only slightly delayed, it still gets processed.
+    Schedules a task with an expiry to prevent duplicate processing when workers fall behind.
+
+    IMPORTANT: Only use for short intervals (<60s). For longer intervals, use crontab schedules instead.
```

Comment:
nobody reads comments :)
we should throw if the interval is longer than 60s

tbh we shojld have this only accept a crontab and convert all the usages

the point of this method was so that people wouldn't forget to add an expiry

if we're adding expiries manually we should fix or delete this :)

Reply:
I'm always up for being stricter 🫡